### ISSUE TITLE
Dodanie TOP 10 użycia schowków

### ISSUE THREAD

**Issue by grabeq123** [Kamilkime-Plugins/KSafe#2]
Dzięki temu pluginowi poprawi sie uczciwość podczas PvP Gildyjnych jak i ogólnych. Schowek ma za zadanie ustawic jakis limit koxow ,refili, perelek na calyn serwerze. Jego zastosowanie jest taki ze gdy gracz ma np 32 refy a na serwie jest ustawiony limit 12 to pozostale 20 refów zostaje odlozone do schowka gdzie gracz bedzie mogł wyplacic je w kazdej chwili lecz gdy uderzy jakiegos gracza lub zje koxa czy refa to nadmiar tych przedmiotów zostanie odlozony do schowka. Plugin najlepiej zrobic w gui gdzie np najechanbiem na refila pokaze sie aktualny stan ile gracz posiada refili w schowku i jaki jest limit na serwerze + gdy kliknie np wyplac wszystko to schowek wyplaci mu wszystkie koxy. Wazna rzecza jest to aby schowek mogl wyplacac wszystko ale tylko wtedy gdy gracz z nikim nie walczy lub nie je koxa bo gdy zje koxa lub bedzie sie z kims klepac to automatycznie zostanie mu ten nadmiar zabrany. Mysle ze sam dodatek jest w miare prosty i nie jest trudny a poprawicie gre dzieki temu wielu osoba . W przyszlosci mozesz dodac do tej wtyczki Topke ze np Top 10 ludzi ktorzy maja najwiecej refow , koxow i perel w schowko fajna sprawa tak jak np top 10 zjedzonych czy uzytych koxow ,refow i perelek. Dzieki temu PvP nabierze inny i lepszy poziom zabawy.

**Comment by Kamilkime** [comment]
> Schowek ma za zadanie ustawic jakis limit koxow ,refili, perelek na calyn serwerze. Jego zastosowanie jest taki ze gdy gracz ma np 32 refy a na serwie jest ustawiony limit 12 to pozostale 20 refów zostaje odlozone do schowka gdzie gracz bedzie mogł wyplacic je w kazdej chwili lecz gdy uderzy jakiegos gracza lub zje koxa czy refa to nadmiar tych przedmiotów zostanie odlozony do schowka. Plugin najlepiej zrobic w gui gdzie np najechanbiem na refila pokaze sie aktualny stan ile gracz posiada refili w schowku i jaki jest limit na serwerze + gdy kliknie np wyplac wszystko to schowek wyplaci mu wszystkie koxy. Wazna rzecza jest to aby schowek mogl wyplacac wszystko ale tylko wtedy gdy gracz z nikim nie walczy lub nie je koxa bo gdy zje koxa lub bedzie sie z kims klepac to automatycznie zostanie mu ten nadmiar zabrany.

No... to jest w zasadzie opis mojego pluginu - to plugin na schowek, więc robi dokładnie to.

> W przyszlosci mozesz dodac do tej wtyczki Topke ze np Top 10 ludzi ktorzy maja najwiecej refow , koxow i perel w schowko fajna sprawa tak jak np top 10 zjedzonych czy uzytych koxow ,refow i perelek.

Ewentualnie to by było coś nowego do dodania

**Comment by CDFN** [comment]
Szczerze to nie widzę zbytnio powiązania między użyciem danych przedmiotów a schowkiem na nie. Jeżeli chodzi o topkę ile kto ma danych zasobów w schowku to mogę dodać w wolnej chwili. Dajcie znać co sądzicie 👀

**Comment by Kamilkime** [comment]
> Szczerze to nie widzę zbytnio powiązania między użyciem danych przedmiotów a schowkiem na nie

No, w pluginie jest task sprawdzający ile masz czego w eq - więc w sumie po zjedzeniu jabłka, jeśli jest odpowiednio ustawiony config, powinno w miarę od razu dodać jabłko ze schowka

**Comment by grabeq123** [comment]
Super sprawa z ta topka dodaj jeszcze limit RzucanychTNT do schowka Razem z TOP 10 osob ktorzy maja ten tnt. Chodzi o to ze w cofnigu bedzie mozna dodac opcje rzucaka np rzucak sie nazywa rzucanetnt wiec w configu wpisujesz jego nazwe i wtedy sie wyswietla. Dodaj tez aby schowek był na wersje 1.8.8. Jakbyś mi podał swojego DC to bym ci wyjasnił dokładnie jak ma to działac ten schowek z FG.

**Comment by grabeq123** [comment]
Dodaj tez aby schowek był na wersje 1.8.8

**Comment by grabeq123** [comment]
Jakbyś mi podał swojego DC to bym ci wyjasnił dokładnie jak ma to działac ten schowek z FG.

**Comment by Kamilkime** [comment]
> Super sprawa z ta topka dodaj jeszcze limit RzucanychTNT do schowka Razem z TOP 10 osob ktorzy maja ten tnt. Chodzi o to ze w cofnigu bedzie mozna dodac opcje rzucaka np rzucak sie nazywa rzucanetnt wiec w configu wpisujesz jego nazwe i wtedy sie wyswietla.

Obecnie plugin pomija nazwę, opis i inne rzeczy w przedmiocie - liczy się tylko typ. Np. ponazywane złote jabłka po trafieniu do schowka i ich wyciągnięciu - są czyste, bez nazwy, opisu, itd. Dodanie czegoś takiego wymagałoby przerobienia całego systemu przechowywania danych... Więc to może na razie nie teraz

> Dodaj tez aby schowek był na wersje 1.8.8.

Wersja 1.0.3 nadal działa z MC 1.8, wersja 1.0.4 wspiera już tylko MC 1.13+ - trzeba by to kiedyś poprzerabiać trochę żeby wspierało obie możliwości na raz

> Jakbyś mi podał swojego DC to bym ci wyjasnił dokładnie jak ma to działac ten schowek z FG.

Najlepiej wszystko pisz tutaj, możesz na kompatybilność z FG zrobić nowe zgłoszenie - lepiej żeby wszystko dotyczące jednego pluginu było w jednym miejscu, mniejszy bałagan

**Comment by grabeq123** [comment]
Po prostu zrób, aby schowek na 1.8.8 był dalej aktualizowany i dodaj te topki :D

**Comment by CDFN** [comment]
>Wersja 1.0.3 nadal działa z MC 1.8, wersja 1.0.4 wspiera już tylko MC 1.13+ - trzeba by to kiedyś poprzerabiać trochę żeby wspierało obie możliwości na raz

A co się sypie z 1.8?

**Comment by Kamilkime** [comment]
> A co się sypie z 1.8?

Tu niestety w drugą stronę - 1.13 ma już tylko Materiale, żeby to dobrze działało to do plugin.yml musiało dojść `api-version` i u niektórych na 1.8 już działać nie chciało, dlatego jest tylko 1.13+ teraz

**Comment by CDFN** [comment]
https://github.com/CryptoMorin/XSeries/
To może pomóc, jakbyś zdecydował się na implementacje to mogę rzucić PR 👍
